ui/ui-components/__tests__/__utils__/ContextProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from "react";
+import { Client } from "../../client/client";
+import { ApiContext } from "../../contexts/api";
+import { GrouparooModelContext } from "../../contexts/grouparooModel";
+import { WebAppContext } from "../../contexts/webApp";
+import { Models } from "../../utils/apiData";
+import { renderNestedContextProviders } from "../../utils/contextHelper";
+
+interface ContextProviderProps {
+  webAppContext?: WebAppContext;
+  client?: Client;
+  model?: Models.GrouparooModelType;
+}
+
+export const ContextProvider: FC<ContextProviderProps> = ({
```

Comment:
Maybe this should have a name that is specific for testing such as `TestContextProvider` so that it's easier to spot that it's not a regular component